File: onebot/src/main/java/cn/evole/onebot/sdk/event/notice/group/GroupEssenceNoticeEvent.java
package cn.evole.onebot.sdk.event.notice.group;

import cn.evole.onebot.sdk.event.notice.NoticeEvent;
import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

/**
 * Description:
 * Author: cnlimiter
 * Date: 2022/10/1 23:30
 * Version: 1.0
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
@EqualsAndHashCode(callSuper = true)
public class GroupEssenceNoticeEvent extends NoticeEvent {
    @SerializedName( "sub_type")
    public String  subType;//add,delete	添加为add,移出为delete
    @SerializedName( "sender_id")
    public Long  senderId	;//消息发送者ID
    @SerializedName( "operator_id")
    public Long operatorId;//操作者ID
    @SerializedName( "message_id")
    public Long messageId	;//消息ID
}

File: onebot/src/main/kotlin/sdk/event/notice/group/GroupEssenceNoticeEvent.kt
package cn.evolvefield.onebot.sdk.event.notice.group

import cn.evolvefield.onebot.sdk.event.notice.NoticeEvent
import com.google.gson.annotations.SerializedName

class GroupEssenceNoticeEvent : NoticeEvent() {
    /**
     * add,delete	添加为add,移出为delete
     */
    @SerializedName("sub_type")
    var subType = ""

    /**
     * 消息发送者ID
     */
    @SerializedName("sender_id")
    var senderId: Long = 0L

    /**
     * 操作者ID
     */
    @SerializedName("operator_id")
    var operatorId: Long = 0L

    /**
     * 消息ID
     */
    @SerializedName("message_id")
    var messageId: Int = 0

    /**
     * 群号
     */
    @SerializedName( "group_id")
    var groupId = 0L
}
